delivery_ups_oca/readme/CONFIGURE.md

```diff
@@ -0,0 +1,17 @@
+To configure this module, you need to:
+
+1.  Add a Shipping Method (delivery.carrier) with provider `ups` and fill in your
+    credentials (UPS Client and UPS Client Secret)
```

Comment:
I suggest to use the updated sentence from [here](https://github.com/OCA/delivery-carrier/pull/991/files#diff-3fd0007ab7b222601a2514695a5a7ae87806c3812eb97d4e0165febae102b1e2).

Reply:
Updated, thank you